force_wfmanager/run.py

```diff
@@ -12,10 +13,22 @@
 from force_wfmanager.wfmanager_plugin import WfManagerPlugin
 
 from traits.api import push_exception_handler
+
+from .version import __version__
+
 push_exception_handler(lambda *args: None, reraise_exceptions=True)
 
 
+@click.command()
+@click.version_option(version=__version__)
+def force_wfmanager():
+    """Launches the workflow manager, including arguments given
```

Comment:
Second gotcha here: click will take the docstring and use it as the summary of the command when invoked with `--help` so you might want to write it with that in mind.  Something like "Launch the FORCE workflow manager application." is probably sufficient here.